region/wreckedship/main/Bowling Alley.json

```diff
@@ -1399,6 +1525,40 @@
         "Morph"
       ],
       "clearsObstacles": ["C"]
+    },
+    {
+      "link": [7, 2],
+      "name": "G-Mode, Bowling Without Going Down",
+      "requires": [
+        {"spikeHits": 1},
+        {"or": [
+          "canInsaneJump",
+          {"spikeHits": 1}
+        ]}
+      ],
+      "flashSuitChecked": true,
+      "note": "Quickly unmorph and jump to the left edge to take as few spike hits as possible."
+    },
+    {
+      "link": [7, 3],
+      "name": "G-Mode, Bowling Then Go Down",
+      "requires": [
+        {"spikeHits": 1},
```

Comment:
You can place a bomb before getting grabbed to bounce upward after the release, avoiding the first spike hit.

Reply:
Very cool, I never would have guessed.

I added a horizontal boost or insane jump to the normal 2-spike strat, it is still in VH, but we could make it a notable if you think it should be moved to expert.